frontend/src/app/testResults/submissions/Submissions.test.tsx

```diff
@@ -132,7 +132,7 @@ const getMocks = () => [
 ];
 
 describe("Submissions", () => {
-  it("should render no results", async () => {
+  it("should render no results and pagination", async () => {
```

Comment:
should this say "should render no results and no pagination"

Reply:
Thank you for your attention to detail!! 🤩 Will fix that up!